packages/semver/src/builders/version/schema.d.ts

```diff
@@ -1,15 +1,47 @@
-import { JsonObject } from '@angular-devkit/core';
-
-export interface VersionBuilderSchema extends JsonObject {
-  dryRun?: boolean;
-  noVerify?: boolean;
-  push?: boolean;
-  remote?: string;
-  baseBranch?: string;
-  syncVersions?: boolean;
-  skipRootChangelog?: boolean;
-  skipProjectChangelog?: boolean;
+export type VersioningType = 'independent' | 'sync-group';
+
+export interface SemverConfig {
+  /**
+   * Name of the project or sync-group, for independent versioning this should match the project name.
```

Comment:
If it's a group, the name should be something explicit as we might need it to run semver on that specific group.

```nx run workspace:version --configs my-group```